src/main.rs

```diff
@@ -59,49 +87,78 @@ pub fn main() {
         };
         if let Some(Command::Sessions(Sessions::Attach {
             mut session_name,
+            create,
             force,
-        })) = opts.command.clone()
-        {
-            if let Some(session) = session_name.as_ref() {
-                assert_session(session);
+        })) = opts.command.clone() {
+
+            let mut sessions = get_sessions();
+            let mut start_new_session = false;
+
+            if session_name.is_none() {
+                if sessions.len() == 1 {
+                    // If a session name was omitted but there is only one session, attach it
+                    session_name = sessions.pop();
+                    println!("Attaching session {:?}", session_name);
+                } else if create {
```

Comment:
This should be checked only if there are no active sessions. If there is only one active session, we attach to it. If there are more than one active sessions we exit with a message. If there are no active sessions, we create one if `create` flag was passed, otherwise exit.

Reply:
Well this is how tmux does it to allow one to create a new fresh session if the previous one had died. I think this is useful.

If we wan't the `<name>` to be compulsory for `zellij attach --create <name>` then that should maybe be checked already in the command-line argument parsing phase.

Reply:
I agree that we should allow users to create a fresh session if one does not exist. But this case here is the one where the user does not mention the session name. Since there is no session name, we will create a new session only if there are no active sessions. Otherwise, `zellij attach --create` will keep on creating a new session each time even though there are more than 1 active sessions and the user might want to attach to one of those.

IMO this is the behaviour that zellij should have with `--create` flag:
1. If session name is specified:
- If a session with name `<session_name>` is present, attach to it.
- If a session with name `<session_name>` does not exist, create a new session.

2. If no session name is specified:
-  If only a single session exists, attach to it.
-  If more than one session exists, exit with a message (and list the sessions).
-  If no session exists, create a new one with a random name

Do you have something else in mind?